Reworking scale section and optimizations (#279)
PrestaShop/docs: src/content/1.7/scale/_index.md

### Chapter 8

> # Scale

Reviewer: Please keep the previous title if you can. I think it's better for SEO :)

> Scaling your PrestaShop instalation is something we hope every merchant will need, because it means her business is growing.

Reviewer: > her business is growing

Alternatively you can use a gender-neutral "their" -> "their business is growing"

> Hopefully, there are plenty advices in these pages that will help.

Reviewer: ```suggestion
Hopefully, there are plenty of advices in these pages that will help.
```

> # Scalings

Reviewer: ```suggestion
## Scalings
```

It's important to have only one h1 on the page.

Before doing anything to your PrestaShop setup, there are several things you need to know.

> First, to be aware there are several ways to scale any application, PrestaShop incuded.

Reviewer: ```suggestion
First, be aware that there are several ways to scale any application, PrestaShop included.
```

> ## Vertical scaling

Reviewer: ```suggestion
### Vertical scaling
```

> The most used one is called vertical scaling, where you boost your server's performance : you add some RAM, some CPU, increase the disks performances, and so on. It can be quick and easy if your hoster allows it and does not require any change on your application setup and configuration.

Reviewer: ```suggestion
The most used one is called vertical scaling, where you boost your server's performance: you add some RAM, some CPU, increase the IO performance, and so on. It can be quick and easy if your hoster allows it and does not require any change in your application setup and configuration.
```

> Though, it's not the more efficient one and doubling the server's resources will harshly double its performances.

Reviewer: ```suggestion
However, this method is not the most efficient one, as doubling the server's resources will not necessarily double its observed performance.
```

> Also, pricing being what it is, vertical scaling may not be linear (depending on your hosting company, your mileage may vary).

Reviewer: I'm not quite sure I understood what you meant:

```suggestion
In addition, the monetary cost of vertical scaling may not be linear. Depending on your hosting company, your mileage may vary.
```

> Doubling your server capacity _may_ cost you more than twice its original price.

Reviewer: ```suggestion
In some cases, doubling your server capacity _may_ cost you more than twice its original price.
```

> ## Horizontal scaling

Reviewer: ```suggestion
### Horizontal scaling
```

> The most efficient way to scale an application remains adding more servers.

Reviewer: If you mean "cost-efficient"

```suggestion
Generally, the most cost-efficient way to scale an application remains adding more servers.
```

If you mean "performance-efficient"

```suggestion
Generally, the most efficient way to scale an application's performance remains adding more servers.
```

> It may not be as simple as it sounds as most PrestaShop file require being shared (or at least synchronized) across the instances, but it's not that hard when you have figured it out.

Reviewer: ```suggestion
It may not be as simple as it sounds, as PrestaShop requires many files being shared (or at least synchronized) across instances, but it's not that hard when you have figured it out.
```

> Also, when you have done the work to add your first server, the other ones should be far more easy.

Reviewer: ```suggestion
Also, once you have set up your first server, setting additional ones should be far easier.
```
